src/containers/DepositModal.tsx

```diff
@@ -1,18 +1,24 @@
-import type { ReactElement } from 'react'
-import { beeDebugApi } from '../services/bee'
+import { ReactElement, useContext } from 'react'
+import { Context as SettingsContext } from '../providers/Settings'
 
 import WDModal from '../components/WDModal'
```

Comment:
This seems to be used in both `DepositModal` and `WithdrawModal` so it seems to me that it would be better to call `WithdrawDepositModal` instead of `WDModal`